perlin.py

```diff
@@ -60,10 +60,10 @@ def sample_edge_noise(edge: sumolib.net.edge.Edge, base: int, centre,
     :return: the value between [0:1]
     """
     x, y = get_edge_pair_centroid(edge.getShape())
-    noise_value = get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves)
-    gradient = (1 - (distance((x, y), centre) / radius)) * centre_weight
+    noise_value = get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves) ** 1.3
```

Comment:
I'm guessing `1.3` is just a voodoo param?

Reply:
Yes, but I actually wanted to remove that again since smoothstep did a better job. Removed in 06e31ce